test/layer/tile_layer/tile_bounds/crs_fakes.dart

```diff
@@ -14,14 +14,16 @@ class FakeInfiniteCrs extends Crs {
 
   /// Any projection just to get non-zero coordiantes.
   @override
-  Point<double> latLngToPoint(LatLng latlng, double zoom) {
-    return const Epsg3857().latLngToPoint(latlng, zoom);
-  }
+  (double, double) latLngToXY(LatLng latlng, double scale) =>
+      const Epsg3857().latLngToXY(latlng, scale);
 
   @override
-  (double, double) latLngToXY(LatLng latlng, double scale) {
-    return const Epsg3857().latLngToXY(latlng, scale);
-  }
```

Comment:
Why has this been removed?

Reply:
It was never needed (there's a default impl) and it's in a test so I removed it while adding impls for the new interfaces.